packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -152,13 +152,14 @@ const Row = styled.div`
 `
 const RightColumn = styled.div`
   width: 40%;
-  margin: 0px 15px;
+  border-radius: 4px;
+  margin-left: 24px;
 
   &:first-child {
     margin-left: 0px;
   }
   &:last-child {
-    margin-right: 0px;
+    margin-right: -24px;
```

Comment:
Would it be possible to achieve this by changing the static width set in DocumentViewer?

https://github.com/opencrvs/opencrvs-core/blob/4b970c36ef456faa55fa32bdce4a1d180d0a661d/packages/components/src/components/interface/DocumentViewer.tsx#L18

Negative margin are sometimes a little prone to unexpected behaviour on different devices